lib/plivo/xml/multipartycall.rb

```diff
@@ -0,0 +1,175 @@
+          raise PlivoXMLError, "role not mentioned : possible values - Agent / Supervisor / Customer"
+        end
+
+        if attributes[:maxDuration] && !VALID_MAX_DURATION_VALUES.include?(attributes[:maxDuration])
+          raise PlivoXMLError, "invalid attribute value #{attributes[:maxDuration]} for maxDuration"
+        elsif !attributes[:maxDuration]
+          attributes[:maxDuration] = 14400
+        end
+
+        if attributes[:maxParticipants] && !VALID_MAX_PARTICIPANT_VALUES.include?(attributes[:maxParticipants])
+          raise PlivoXMLError, "invalid attribute value #{attributes[:maxParticipants]} for maxParticipants"
+        elsif !attributes[:maxParticipants]
+          attributes[:maxParticipants] = 10
+        end
+
+        if attributes[:waitMusicMethod] && !VALID_WAIT_MUSIC_METHOD_VALUES.include?(attributes[:waitMusicMethod])
+          raise PlivoXMLError, "invalid attribute value #{attributes[:waitMusicMethod]} for waitMusicMethod"
+        elsif !attributes[:waitMusicMethod]
+          attributes[:waitMusicMethod] = 'GET'
+        end
+
+        if attributes[:agentHoldMusicMethod] && !VALID_AGENT_HOLD_MUSIC_METHOD_VALUES.include?(attributes[:agentHoldMusicMethod])
+          raise PlivoXMLError, "invalid attribute value #{attributes[:agentHoldMusicMethod]} for agentHoldMusicMethod"
+        elsif !attributes[:agentHoldMusicMethod]
+          attributes[:agentHoldMusicMethod] = 'GET'
+        end
+
+        if attributes[:customerHoldMusicMethod] && !VALID_CUSTOMER_HOLD_MUSIC_METHOD_VALUES.include?(attributes[:customerHoldMusicMethod])
+          raise PlivoXMLError, "invalid attribute value #{attributes[:customerHoldMusicMethod]} for customerHoldMusicMethod"
+        elsif !attributes[:customerHoldMusicMethod]
+          attributes[:customerHoldMusicMethod] = 'GET'
+        end
+
+        if attributes[:record] && !VALID_RECORD_VALUES.include?(attributes[:record])
+          raise PlivoXMLError, "invalid attribute value #{attributes[:record]} for record"
+        elsif !attributes[:record]
+          attributes[:record] = false
+        end
+
+        if attributes[:recordFileFormat] && !VALID_RECORD_FILE_FORMAT_VALUES.include?(attributes[:recordFileFormat])
+          raise PlivoXMLError, "invalid attribute value #{attributes[:recordFileFormat]} for recordFileFormat"
+        elsif !attributes[:recordFileFormat]
+          attributes[:recordFileFormat] = 'mp3'
+        end
+
+        if attributes[:recordingCallbackMethod] && !VALID_RECORDING_CALLBACK_METHOD_VALUES.include?(attributes[:recordingCallbackMethod])
+          raise PlivoXMLError, "invalid attribute value #{attributes[:recordingCallbackMethod]} for recordingCallbackMethod"
+        elsif !attributes[:recordingCallbackMethod]
+          attributes[:recordingCallbackMethod] = 'mp3'
+        end
+
+        if !attributes[:statusCallbackEvents]
+          attributes[:statusCallbackEvents] = 'mpc-state-changes,participant-state-changes'
+        end
```

Comment:
add new validations for this field
if value when split by `COMMA` is among the values given in spec